src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,15 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
can we use only `flatMap` here?
```suggestion
                .flatMap(string -> Arrays.stream(string.split(",")))
```

Reply:
done